### DISCUSSION TITLE
Request: Drag map with middle mouse click instead of right click

### DISCUSSION THREAD

**Discussion by webfischi** [Keriew/augustus#257] [Ideas]
It would be great to have the middle mouse button (clicking the scroll wheel) to drag the map, like in Frostpunk. Especially because right click is already used to open object information panels.

A checkbox in the settings to switch between right and middle mouse button would be great as well, for those who prefer using the right one.

**Comment by xyn22** [comment]
I second that, also with two-fingertip-drag gestures on laptops touchpads.

**Comment by crudelios** [comment]
I like this idea. @bvschaik, what do you think about this for Julius - instead of disable right click, we could change the setting to use middle button instead of right button?

**Comment by crudelios** [comment]
This is also related to bvschaik/julius#567

**Comment by crudelios** [comment]
> I second that, also with two-fingertip-drag gestures on laptops touchpads.

This is not possible as touchpads are viewed as a mouse and multi-finger gestures as mouse scrolling.
